impl/src/main/java/space/npstr/magma/impl/connections/AudioWebSocket.java

```diff
@@ -317,6 +318,12 @@ private Disposable connect(final ClosingWebSocketClient client, final URI endpoi
                 .subscribe();
     }
 
+    // see https://discordapp.com/developers/docs/reference#user-agent
+    private String getUserAgent() {
+        String version = this.magmaVersionProvider.getVersion();
+        return String.format("DiscordBot (%s, %s)", "https://magma.beer", version);
```

Comment:
Is there a reason to not make this a constant?

Reply:
Actually the whole user agent can be a constant, there does not seem to be a need to calculate it each time.